notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2663 @@
+{
```

Comment:
<p>I think it's worth clarifying that specifically it's power laws that will be straight lines on log-log axes:</p><p>y = C x^a</p><p>log(y) = a log(x) + log(C)</p><p> </p><p>So the slope of the line in the log-log plot is the exponent of the power law.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>